activities/index.md

```diff
@@ -2,6 +2,9 @@
 # SPDX-License-Identifier: CC0-1.0
 # SPDX-FileCopyrightText: 2018-2023 The Foundation for Public Code <info@publiccode.net>
 type: Index
+redirect _from:
```

Comment:
I think the redirects here are good. Perhaps we should add a line at the top like "If you were looking for another activity but ended up here, we have stopped doing that activity."

Reply:
Moved to #1350